src/rpc/blockchain.cpp

```diff
@@ -313,6 +402,30 @@ UniValue getrawmempool(const UniValue& params, bool fHelp)
     return mempoolToJSON(fVerbose);
 }
 
+// insightexplorer
+UniValue getblockdeltas(const UniValue& params, bool fHelp)
+{
+    if (fHelp || params.size() != 1)
+        throw runtime_error("");
```

Comment:
Is this the documentation for this RPC? >.<

Reply:
I see this is added in #3972.

Reply:
I wish I'd noticed your comment before I implemented this -- again! Oh well, I think I did it better this time.